runtime/vm/JVMImage.hpp

```diff
@@ -26,20 +26,37 @@
 #ifndef JVMIMAGE_HPP_
 #define JVMIMAGE_HPP_
 
-#include "j9_types.h"
+#include "j9.h"
+#include "j9comp.h"
+
+extern "C" 
```

Comment:
same as above probably going to move this to an api file

Reply:
vm_api.h is a good place to put it

Reply:
ok. Might move in the future to jvmimage_api.h and place that file in j9protos